lib/src/eval/shared/stdlib/core/map.dart

```diff
@@ -124,6 +136,18 @@ class $Map<K, V> implements Map<K, V>, $Instance {
     return target;
   }
 
+  static const $Function __containsKey = $Function(_containsKey);
+
+  static $Value? _containsKey(
+      Runtime runtime, $Value? target, List<$Value?> args) {
+    final mapObject = target!.$reified as Map;
+    // remove "$" and double quotation sign(")
+    final keyToFind =
+        args[0].toString().replaceAll('\$', '').replaceAll('"', '');
```

Comment:
Instead of using `$reified`, use `$value` to get the map. Then you can remove the whole toString / replaceAll chain and just do `containsKey(args[0])` directly. This will be much faster and less error-prone.